examples/player/simid_player.html

```diff
@@ -23,15 +23,30 @@
     }
   </style>
   <script>
+    //if ad is non-linear allow user to insert specific dimensions
+    function enableDisable() {
+      let isNonLin = document.getElementById("nonlinear_ad");
```

Comment:
In general, avoid abbreviations

https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers

`isNonLinear`

But also, this variable is only the selector element, calling something prefixed with `is` makes it sound like its a boolean.  I would either make this variable the actual boolean, or name it something to indicate that it is an element

Reply:
in the toggle function that replaces this in the other PR I don't even save the variable so this is fixed